src/middlewared/middlewared/alert/source/m_series_old_bios_version_freebsd.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding=utf-8 -*-
+import logging
+
+from middlewared.alert.base import Alert, AlertClass, AlertCategory, AlertLevel, AlertSource
+
+logger = logging.getLogger(__name__)
+
+
+class TrueNASMOldBIOSVersionAlertClass(AlertClass):
+    category = AlertCategory.HARDWARE
+    level = AlertLevel.EMERGENCY
+    title = "Old BIOS Version"
+    text = (
+        "Your M-Series TrueNAS has an old BIOS version. "
+        "Please contact support immediately."
```

Comment:
@Mrt134 @wmohanlon same question here, please

Reply:
Can we lose the word "immediately", to soften it a bit?  I don't want to downplay the importance, but I don't want support to get hammered, either.